dr-script/common.sh

```diff
@@ -32,9 +32,7 @@ HC_CLUSTER_ID=""
 HC_CLUSTER_NS=""
 HC_REGION=""
 HC_CLUSTER_NAME=""
-HC_CLUSTER_DIR="${BASE_PATH}/${HC_CLUSTER_NAME}"
-HC_KUBECONFIG="${HC_CLUSTER_DIR}/kubeconfig"
-BACKUP_DIR=${HC_CLUSTER_DIR}/backup
+HCPASS=""
```

Comment:
```suggestion
HC_PASS=""
```
I think we should update the HCPASS to HC_PASS just the same as other `HC_` vars. 
I should've done it in the last PR. My bad.

Reply:
Ok, let me do that for now, but I'm hoping to automate and not require this eventually.